synchronicity/synchronizer.py

```diff
@@ -476,9 +477,15 @@ def _wrap_class(self, cls, interface, name):
     def _wrap_class_or_function(self, object, interface):
         if _WRAPPED_ATTR not in object.__dict__:
             setattr(object, _WRAPPED_ATTR, {})
+
         interfaces = object.__dict__[_WRAPPED_ATTR]
         if interface in interfaces:
+            if self._multiwrap_warning:
```

Comment:
I assumed we wanted to warn about multiple wrap calls on functions even if they weren't re-wrapped as a way to signal to users that they are calling synchronicity multiple times on the same functions. Since the wrapped async function is no longer inspect.isfunction() the inner multiwrap would no longer trigger for those functions so I moved it out here.

We could just remove it if it's not needed? I wasn't sure what the purpose of the warning was :)

Reply:
ok, i don't think this quite makes sense, but otoh i think i want to remove the multiwrap thing anyway, so it's fine